docs/reference/formatter_widget.rst

```diff
@@ -127,81 +126,27 @@ If you stop here, the most interesting part will not be present. Let's edit some
     If you want to see an actual implementation of this can, you can have a look at the ``SonataNewsBundle``.
 
 
-Dynamic Input
```

Comment:
Why are you removing this block?

Reply:
IMO it is duplicate and the same to achieve both

Reply:
all the stuff is already named in the installation section, so it is duplicate content
https://github.com/sonata-project/SonataFormatterBundle/blob/3.x/docs/reference/installation.rst#configuration